core/modules/dblog/css/dblog.css

```diff
@@ -68,18 +68,22 @@
   width: 30px;
   height: 30px;
   border-radius: 15px;
+  background: #ccc url(../../../misc/message-32-info.png) no-repeat center center;
+  background-size: 16px;
```

Comment:
I think that the changes in this file are overlapping with https://github.com/backdrop/backdrop/pull/2426/files#diff-617c1d5b7fa1ac6b8db6e8b4ad9db7e9 (for https://github.com/backdrop/backdrop-issues/issues/3437)